themes/_core/js/product.js

```diff
@@ -169,7 +169,7 @@ function updateProduct(event, eventType, updateUrl) {
 
         $('.quickview .product-prices, .page-product:not(.modal-open) .row .product-prices').replaceWith(data.product_prices);
         $('.quickview .product-customization, .page-product:not(.modal-open) .row .product-customization').replaceWith(data.product_customization);
-        $('.quickview .product-variants .page-product:not(.modal-open) .row .product-variants').replaceWith(data.product_variants);
+        $('.quickview .product-variants .page-product:not(.modal-open) .row .product-variants, .page-product:not(.modal-open) .product-information .product-variants').replaceWith(data.product_variants);
```

Comment:
How come the previous selector didn't work any more? Did the markup changed? Does it mean the other selector could potentially not work as well?

Reply:
By other selectors you mean the other product-variants selectors ?
Actually, I do reproduce the quickview context but I dont know if there are other ways to display it

And all the other selectors had quickview and classic page selectors except "product-variants"

Reply:
Ohhh I think I see the issue, seems like there was a missing `,`
```suggestion
        $('.quickview .product-variants, .page-product:not(.modal-open) .row .product-variants').replaceWith(data.product_variants);
```

Reply:
DOne, and you were right, there was just a missing comma